daemon.go

```diff
@@ -73,16 +76,17 @@ import (
 	"path/filepath"
 	"regexp"
 	"strings"
-	"syscall"
 	"time"
 
 	"github.com/fatih/color"
-	"github.com/fsnotify/fsnotify"
 )
 
 // Milliseconds to wait for the next job to begin after a file change
 const WorkDelay = 900
 
+// Milliseconds of interval between polling file changes when polling option is selected
+const PollingInterval = 100
```

Comment:
Is this something that should be an option? Did you have to tune this often?

Reply:
I have tested folder with about 150 files only, so there might be a problem with thousands of files. It would be better to make this an optional parameter.